bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/BlueZAdapterConstants.java

```diff
@@ -1,33 +0,0 @@
-/**
```

Comment:
You shouldn't delete this file, instead you rename your dbus config file to this one instead.